### PR TITLE
Jarv/clean ami

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -72,7 +79,7 @@ if [[ -z $ami ]]; then
 fi
 
 if [[ -z $instance_type ]]; then
-  instance_type="m1.medium"
+  instance_type="m3.medium"
```
Comment (hkim823) [inline util/jenkins/ansible-provision.sh]: I thought we decided to go back to m1.medium as the default for more storage for db clones?

Comment (jarv) [inline util/jenkins/ansible-provision.sh]: fixed

File: util/jenkins/ansible-provision.sh [hunk 2]
```diff
@@ -133,6 +144,23 @@ instance_tags:
...
+EDXAPP_PREVIEW_LMS_BASE: preview.${deploy_host}
+EDXAPP_LMS_BASE: ${deploy_host}
+EDXAPP_CMS_BASE: studio.${deploy_host}
+EDXAPP_SITE_NAME: ${deploy_host}
+CERTS_DOWNLOAD_URL: "http://${deploy_host}:18090"
+CERTS_VERIFY_URL: "http://${deploy_host}:18090"
+edx_internal: True
 COMMON_USER_INFO:
   - name: ${github_username}
     github: true
```
Comment (hkim823) [inline util/jenkins/ansible-provision.sh:166]: Can we move COMMON_USER_INFO to Global Vars?  That way even if someone spins up Ubuntu 12.04 base, it puts their Github ssh keys to be able to log in without having to run the separate job.

**Comment by hkim823** [comment]
i'm fine with merging it in as is

**Pull request merged**